.github/workflows/plugins.yml

```diff
@@ -56,36 +65,46 @@ jobs:
             port: "8080"
           - version: "5.0"
             port: "8080"
+        ansible:
+          # - stable-2.9 # Only if your collection supports Ansible 2.9
+          - stable-2.10
+          #- devel
+        python:
+          - 2.7
+          - 3.7
+          #- 3.8 blocked by https://github.com/ansible/ansible/pull/70155
 
     steps:
       - name: Check out code
-        uses: actions/checkout@v1
+        uses: actions/checkout@v2
         with:
           path: ansible_collections/community/zabbix
 
-      - name: Set up Python 3.6
+      - name: Set up Python ${{ matrix.ansible }}
         uses: actions/setup-python@v1
         with:
-          python-version: 3.6
+          python-version: ${{ matrix.python }}
+
+      - name: Install ansible-base (${{ matrix.ansible }})
+        run: pip install https://github.com/ansible/ansible/archive/${{ matrix.ansible }}.tar.gz --disable-pip-version-check
 
       - name: Install dependencies
         run: pip install docker-compose zabbix-api
 
-      - name: Install ansible-base (stable-2.10)
-        run: pip install https://github.com/ansible/ansible/archive/stable-2.10.tar.gz --disable-pip-version-check
-
       - name: Zabbix container server provisioning
         run: docker-compose up -d
+        working-directory: ./ansible_collections/community/zabbix
         env:
           zabbix_version: ${{ matrix.zabbix_container.version }}
           zabbix_port: ${{ matrix.zabbix_container.port }}
 
       - name: Run integration test
-        run: ansible-test integration -v --color --retry-on-error --continue-on-error --diff --requirements --coverage
+        run: ansible-test integration -v --color --retry-on-error --continue-on-error --diff --python ${{ matrix.python }} --requirements --coverage
```

Comment:
Any reason to run sanity in docker but this one in the VM directly?

Reply:
Hi @webknjaz and yes, we run integration tests against multiple versions of Zabbix servers (haven't found prettier solution via Github Actions than "stack"s being deployed by [docker-compose](https://github.com/ansible-collections/community.zabbix/blob/master/docker-compose.yml)) as we are not interested in testing on any major OS distro since we should be OS independent.

Getting each component of Zabbix working together was a real pain and this was far simpler solution since we are use this easily for local development too.

Reply:
I meant the missing `--docker` option here.

Reply:
my bad for misinterpreting. Result would be non-routable network between test container and zabbix stack if I recall correctly

[See @gundalow comment](https://github.com/ansible-collections/community.zabbix/issues/4#issuecomment-599468228)

Reply:
Ah, I see. I wonder if using GHA native services would work: https://help.github.com/en/actions/configuring-and-managing-workflows/about-service-containers#mapping-docker-host-and-service-container-ports

Reply:
Heh, I've already tried that in the same issue, although I don't recall much about it now - [comment](https://github.com/ansible-collections/community.zabbix/issues/4#issuecomment-602087010)

Reply:
@D3DeFi it is now possible to use `--docker` if you synchronize with the newest collection template workflow @ https://github.com/ansible-collections/collection_template/blob/main/.github/workflows/ansible-test.yml that uses a unified action solving this problem.